app/src/main/res/values/strings.xml

```diff
@@ -245,4 +245,10 @@
   <string name="about_rate_us"><u>Rate Us</u></string>
   <string name="about_faq">Frequently Asked Questions</string>
   <string name="welcome_skip_button">Skip Tutorial</string>
+  <string name="about_translate"><u>Translate</u></string>
```

Comment:
The <u> things are not working in translatewiki.net at the moment unless escaped.